unison-src/transcripts/fix3265.output.md

```diff
@@ -33,19 +33,20 @@ are three cases that need to be 'fixed up.'
           ⧩
           Any
             (w x ->
-              use Nat + drop
-              f1 y =
-                match y with
-                  0 -> w + x
-                  n -> 1 + f0 (drop y 1)
-              f0 y =
-                match y with
-                  0 -> x
-                  n -> 1 + f1 (drop y 1)
-              f2 x = f2 x
-              f3 x y = 1 + y + f2 x
-              g h = h 1 + x
-              g (z -> x + f0 z))
+              let
+                use Nat + drop
+                f1 y =
+                  match y with
+                    0 -> w + x
+                    n -> 1 + f0 (drop y 1)
+                f0 y =
+                  match y with
+                    0 -> x
+                    n -> 1 + f1 (drop y 1)
+                f2 x = f2 x
+                f3 x y = 1 + y + f2 x
+                g h = h 1 + x
```

Comment:
This is the difference in output. Note the introduction of `let` at the start of the block.